site/content/concepts/command-and-control.md

```diff
@@ -66,3 +66,31 @@ The response of the command will be sent by the device to
 If the device is authenticated, the `tenant` and `device-id` are left empty (resulting in 3 subsequent `/`s).
 
 ![Command & Control over MQTT Adapter](../command_control_concept_mqtt.png) 
+
+## Command & Control over AMQP Adapter
+
+When a device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link specifying the following source address:
+
+* `control` for authenticated devices
+* `control/tenant/device-id` for unauthenticated devices
+
+Once the receiver link is opened, the AMQP adapter sends command messages to devices through the link. The `subject` property of the message contains the actual command to be executed on the device while the `<request-id>`,used to correlate the command request with the response, is specified as an application property under key `APP_PROPERTY_REQUEST_ID`.
```

Comment:
property names should be marked up using italics (i.e. wrapping them in asterisks). 
I do not think that the name of the property to convey the request ID is *APP_PROPERTY_REQUEST_ID*, or is it?

Reply:
welcome back @sophokles73. This is already a good start :+1: 

> I do not think that the name of the property to convey the request ID is APP_PROPERTY_REQUEST_ID, or is it?

I defined it as a constant in the `AmqpAdapterConstants.java` class. I think its better to omit mentioning the application key property from the docs, not so?

Reply:
no